stories/Typography/Typography.story.jsx

```diff
@@ -1,8 +1,10 @@
+/* eslint-disable */
```

Comment:
Can you enable and prettify the file?

Reply:
tried and didn't work for some reason, even tried running prettier manually